Makefile

```diff
@@ -97,13 +105,13 @@ $(BIN)/protoc: | $(BIN)
 	rm $(BIN)/protoc.zip
 
 # any generated file - they all depend on each other / are generated at once, so any will work
-PROTO_GEN_SRC = .gen/proto/admin/v1/service.pb.go
+PROTO_GEN_SRC = ./.gen/proto/admin/v1/service.pb.go
```

Comment:
makes this consistent with the `find` results, so `sort` will put them where they belong / deduplicate.